design_docs/config-file_reorganization.md

```diff
@@ -0,0 +1,108 @@
+
+An analogous approach will also be added at the command line, for example:
+```
+./run_analysis.py config.analysis --generate all,no_ocean,all_timeseries
+```
+(I am open to other syntax.)  If the --generate flag is used on the command line, it will replace the generate option in the config file.
+
+As an aside, I note that it is not clear if future analysis modules will fit neatly into categories like "time series" and "model vs. observations", and these categories are not meant to be all-encompassing.
+
+<h2> Implementation: there should be a simplified template for config files <br>
+Date last modified: 2017/01/29 <br>
+Contributors: Xylar Asay-Davis
+</h2>
+
+Such a template has been implemented in #86.  A subdirectory `configs` will be added with several examples from runs on LANL IC and on Edison at NERSC.  Other examples can be added as appropriate and useful.
+
+<h2> Implementation: removal of ACME specific config options <br>
+Date last modified: 2017/01/29 <br>
+Contributors: Xylar Asay-Davis
+</h2>
+
+This item needs some discussion.  in #86, I would like to renamed `ref_casename_v0` to simply `ref_casename`.  `casename` and `ref_casename_v0` are used for file names and figure titles (essentially legends).  It would be useful to discuss what the relevant equivalents would be fore standalone MPAS runs.
+
+<h1> Testing </h1>
+
+<h2> Testing and Validation: a simple way of turning on and off individual analysis modules <br>
+Date last modified: 2017/01/29 <br>
+Contributors: Xylar Asay-Davis
+</h2>
+
+This will be difficult to test with CI since we currently don't have CI that can perform tests with `run_analysis.py`.
+
+Instead, I will manually test a variety of combinations of `generate` lists (both in the config file and on the command line).  I will list the tests I perform in #86.
+
+<h2> Testing and Validation: there should be a simplified template for config files <br>
+Date last modified: 2017/01/29 <br>
+Contributors: Xylar Asay-Davis
+</h2>
+
+There is not a way to test the template in the usual sense.  Instead, the test will be asking other developers and users to adapt the template for new runs to make sure it is intuitive.
+
+<h2> Testing and Validation: removal of ACME specific config options <br>
+Date last modified: 2017/01/29 <br>
+Contributors: Xylar Asay-Davis
+</h2>
+
+The analysis will be tested on both MPAS and ACME runs to make sure the resulting image files have useful file names and titles (legends).  
```

Comment:
How will you assess success for this goal?  Naming is a subjective process, at least for the best names.  This seems like something that will require some group discussion.